docs/changes/cosmology/17702.api.rst

```diff
@@ -0,0 +1,2 @@
+A new public module, ``astropy.cosmology.traits``, has been added to provide building
+blocks for custom cosmologies.
```

Comment:
You haven't mentioned which traits are available, but that also means that if you manage to make any more traits public before the next release then the change log does not have to be updated, so in this case that's a good thing.

Reply:
Won't happen before v7.1 is out.

Reply:
Unfortunately it won't happen before v7.1 is out.
I enumerated the currently available traits.